db/migrate/20210112182218_create_bitflyer_connection.rb

```diff
@@ -0,0 +1,27 @@
+class CreateBitflyerConnection < ActiveRecord::Migration[6.0]
+  def change
+    create_table :bitflyer_connections, id: :uuid, default: -> { "uuid_generate_v4()"}, force: :cascade do |t|
+      t.belongs_to :publisher, type: :uuid, index: true, null: false
+
+      t.string :encrypted_access_token
+      t.string :encrypted_access_token_iv, index: { unique: true }
+
+      t.string :encrypted_refresh_token
+      t.string :encrypted_refresh_token_iv, index: { unique: true }
+
+      t.string :expires_in
+      t.datetime :access_expiration_time
+      t.string :display_name
+      t.string :state_token
+
+      t.string :scope
+
+      t.string :status, index: true
```

Comment:
Nit: Is there a query that's heavily dependent on `status`? If not, I don't think we should index this.